### ISSUE TITLE
W3C Workshop on Wide Color Gamut and High Dynamic Range for the Web 

### ISSUE THREAD

**Issue by svgeesus** [w3c/strategy#230]
Strat has for some time wanted to organize a workshop on Wide Color Gamut (WCG) and High Dynamic Range (HDR) for the Web. Due to slower than expected/more complex than expected issues adding WCG to the Web, this was pushed forward. However, the [ColorWeb CG report](https://w3c.github.io/ColorWeb-CG/) is maturing and will be discussed at TPAC. Browsers are now actively implementing. Houdini is looking at a WCG Color object for the typed OM. So (to allow time to get horizontal review, approvals, speakers) I propose a workshop for Spring 2021.

See draft [Call for Participation](https://www.w3.org/Graphics/Color/Workshop/) for likely scope and content. This is still early stage, but the topics are all ripe for exploration.

I have four confirmed and two more potential program committee members.

Horizontal review:

- [x] [A11y](https://github.com/w3c/strategy/issues/230#issuecomment-697637565)
- [x] [I18n](https://github.com/w3c/strategy/issues/230#issuecomment-701928711)
- [x] [Privacy](https://github.com/w3c/strategy/issues/230#issuecomment-702299622)
- [x] [Security](https://github.com/w3c/strategy/issues/230#issuecomment-702297048)

**Comment by svgeesus** [comment]
The [GitHub repo](https://github.com/w3c/web-wcg-hdr-workshop) for the workshop

**Comment by svgeesus** [comment]
(inadvertent closure)

**Comment by michael-n-cooper** [comment]
APA has no comments on the workshop proposal, but is interested in the subject matter and thinks @myndex might be interested to participate in this. Over to @brewerj to complete accessibility horizontal review.

**Comment by Myndex** [comment]
> APA has no comments on the workshop proposal, but is interested in the subject matter and thinks @Myndex might be interested to participate in this. Over to @brewerj to complete accessibility horizontal review.

Thank you @michael-n-cooper   and Hello  @svgeesus  — this does look interesting, and certainly some forms of impairment (particularly protanopia) are expected to have problems with physical display devices using rec2020 or rec2100...

Let me know if I can help/where to participate.

Thank you

Andy

**Comment by svgeesus** [comment]
Thank you @michael-n-cooper for the swift response from APA side

@Myndex yes, increased observer metamerism with pure spectral primaries as used in BT.2020 & BT.2100, and the need for individual observer functions rather than a single standard observer to mitigate this, is totally in scope. Would love to have you participate.

**Comment by himorin** [comment]
(sorry dropped from my todo list...) no comment/suggestion raised in i18n group.

**Comment by samuelweiler** [comment]
The CFP mentions wanting security people, but it's not clear to me what the draw - or need - for them is here.  Perhaps the CFP could explain the problem space better, to motivate their interest?

**Comment by samuelweiler** [comment]
The CFP says "Detecting ... HDR support..." and does not have a parallel item under WCG - is that intentional?  Do you somehow view WCG support as not needing detection?

Any time I see detection or negotiation of capabilities, I worry about browser fingerprinting.  As you proceed, contemplate ways to avoid that negotiation.  Are there formats that gracefully downgrade ("are backwards compatible"), so the same image(s) can be sent to all clients, no matter their feature set, and the server never needs to know what the client supports?  For more info: https://w3c.github.io/fingerprinting-guidance/

Is it worthwhile adding "seeking formats that are backward compatible" to the agenda?

**Comment by svgeesus** [comment]
@samuelweiler thanks for your thoughtful comments.

> The CFP says "Detecting ... HDR support..." and does not have a parallel item under WCG - is that intentional? Do you somehow view WCG support as not needing detection? 

No, but we [already have a way to detect WCG support](https://drafts.csswg.org/mediaqueries-5/#color-gamut). While I have heard requests for better or more fine-grained HDR support, so I added that as in-scope although [we do have a recently added way for HDR](https://drafts.csswg.org/mediaqueries-5/#dynamic-range).

**Comment by svgeesus** [comment]
>  Are there formats that gracefully downgrade ("are backwards compatible"), so the same image(s) can be sent to all clients, no matter their feature set, and the server never needs to know what the client supports?

To a certain extent. The HLG transfer function is designed to be "somewhat" backwards compatible with SDR and to be usable without re-rendering in a range of different viewing conditions (dark room to bright daylight). But you would still get a better result from separate SDR and HDR content. In contrast the PQ transfer function is not SDR compatible, uses absolute luminance and can only be viewed in the standard (very dark room) viewing environment.

**Comment by svgeesus** [comment]
> Is it worthwhile adding "seeking formats that are backward compatible" to the workshop scope?

It is worth adding that as a topic, certainly. It would be hard to make it a requirement though.

**Comment by svgeesus** [comment]
> The CFP mentions wanting security people, yet it's not clear to me what the draw - or need - for them is here. Perhaps the CFP could explain the problem space better, to motivate their interest?

To be honest I added security just in case someone wanted to talk about it. I don't have specific security concerns I wanted to discuss.

**Comment by svgeesus** [comment]
Horizontal review is almost complete. @brewerj any additional comments?

@wseltzer is this ready to be taken to w3m or would you like to see more discussion or more materials (if the latter, what?)

**Comment by svgeesus** [comment]
Discussed on [17 Nov 2020 strat call](https://www.w3.org/2020/11/17-strat-minutes.html#item02). Decision taken to request W3M approval.

**Comment by brewerj** [comment]
@svgeesus this looks good, thanks for the attention to accessibility at the early stage. Two minor comments: I suggest asking for @slhenry 's input from the AGWG Low Vision Task Force as you develop this further, even though you already have @michael-n-cooper 's interest on behalf of AGWG itself. 
Also, for your speaker notes, I suggest linking to [How to Make Your Presentations Accessible to All](https://www.w3.org/WAI/teach-advocate/accessible-presentations/
) as an additional resource for presenters, even though you have some good starter notes on how to do accessible presentations. 
Thanks

**Comment by svgeesus** [comment]
Thanks for the feedback @brewerj 
I added the link you suggested to the speaker notes.
Does [AGWG Low Vision TF](https://www.w3.org/groups/tf/low-vision-a11y-tf) not have a GitHub repo?

**Comment by svgeesus** [comment]
Workshop approved at 18 December W3M

**Comment by slhenry** [comment]
> Does AGWG Low Vision TF not have a GitHub repo?

AGWG Low Vision TF is currently inactive. We can send messages to the list, and previous participants will get it.

public-low-vision-a11y-tf@w3.org

**Comment by wseltzer** [comment]
Workshop planned. Homepage: https://www.w3.org/Graphics/Color/Workshop/

**Comment by svgeesus** [comment]
[Final Report](https://www.w3.org/Graphics/Color/Workshop/report.html) and [next steps](https://www.w3.org/Graphics/Color/Workshop/report.html#next)
